### ISSUE TITLE
SIGSEGV after startup, Debian 10, 32 bit

### ISSUE THREAD

**Issue by avodonosov** [Clozure/ccl#274]
Soon after starting CCL on Debian 10, 32bit, I have the below error.

It may happen seconds after startup, or a minute, or 4 minutes. It seems happening in some background thread.

Reproduced for CCL 1.11.5, CCL 1.10.

Rebuild of kernel doesn't help.

In case fix not possible soon, is there any workaround I can use?

If necessary, I can provide ssh access to the server where it happens, your public key will be needed for that.

```
? Unhandled exception 11 at 0x47690f, context->regs at #xbfb0fc28
Exception occurred while executing foreign code
 at _SPffcall + 87
received signal 11; faulting address: (nil)
unexpected si_code value: 128
? for help
[20245] Clozure CL kernel debugger: bt
current thread: tcr = 0xb7c19700, native thread ID = 0x4f15, interrupts enabled


(#xB7B97F7C) #x10429F35 : #<Function %NANOSLEEP #x10429C66> + 719
(#xB7B97FBC) #x104374E5 : #<Function HOUSEKEEPING-LOOP #x10437346> + 415
(#xB7B97FDC) #x10437A2D : #<Function (:INTERNAL (TOPLEVEL-FUNCTION (LISP-DEVELOPMENT-SYSTEM T))) #x1043794E> + 223
[20245] Clozure CL kernel debugger: Current Thread Context Record (tcr) = 0xb7c19700
Control (C) stack area:  low = 0xbf9ad000, high = 0xbfb10104
Value (lisp) stack area: low = 0xb7a4c000, high = 0xb7b98000
Exception stack pointer = 0xbfb10078

```

**Comment by avodonosov** [comment]
Signal 11 is SIGSEGV, so if I understand correctly, the thread was sleeping in `nanosleep` call and was interrupted by the segmentation fault. 

What causes the segmentation fault I don't know. You should not have any custom code running to trigger the error; just waiting in REPL is enough.

**Comment by xrme** [comment]
@avodonosov did the fix for #306 solve this?

**Comment by avodonosov** [comment]
On the server where my error happened, it was recently found that memory was reduced by mistake. After the memory was increased again the error doesn't happen anymore.

So I can not say if it was related to PIE.
